ckanext/collaborators/views/package.py

```diff
@@ -0,0 +1,91 @@
+from flask.views import MethodView
+import ckan.plugins.toolkit as toolkit
+
+from ckan.common import _, g
+import ckan.model as model
+import ckan.lib.navl.dictization_functions as dictization_functions
+import ckan.logic as logic
+
+
+
+def collaborators(dataset_id):
+    context = {u'model': model, u'user': toolkit.c.user}
+    data_dict = {'id': dataset_id}
+    
+    # needed to ckan_extend package/edit_base.html
+    try:    
+        g.pkg_dict = toolkit.get_action('package_show')(context, data_dict)
+    except (toolkit.NotAuthorized, toolkit.ObjectNotFound):
```

Comment:
In this case I think we should show separate errors if the user is not authorized or the dataset is not found (eg 403 and 404)

Reply:
Same with the other view functions